packages/rdbg/rdbg.1.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "erwan.jahier@imag.fr"
+bug-reports: "erwan.jahier@imag.fr"
+authors: [ "Erwan Jahier" ]
+license: "GPL-3"
+homepage: "http://rdbg.forge.imag.fr/"
+dev-repo: "https://forge.imag.fr/anonscm/git/rdbg/rdbg.git"
+build: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocamlfind" "remove" "rdbg-plugin"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "base-unix"
+  ("camlp4" {= "4.03.0"} | "camlp4" {>= "4.02.1+system"} | "camlp4" {= "4.02.1+2"} | "camlp4" {= "4.02.1+1"} | "camlp4" {= "4.02.1"} | "camlp4" {= "4.02.0-1modular-implicits"} | "camlp4" {= "4.02.0+system"} | "camlp4" {= "4.02.0+2"} | "camlp4" {= "4.02.0+1"} | "camlp4" {= "4.02.0"} | "camlp4" {= "4.01.0"})
```

Comment:
On Wed, 12 Aug 2015 06:41:00 -0700, Anil Madhavapeddy wrote:
> 
> > +  ("camlp4" {= "4.03.0"} | "camlp4" {>= "4.02.1+system"} | "camlp4" {= "4.02.1+2"} | "camlp4" {= "4.02.1+1"} | "camlp4" {= "4.02.1"} | "camlp4" {= "4.02.0-1modular-implicits"} | "camlp4" {= "4.02.0+system"} | "camlp4" {= "4.02.0+2"} | "camlp4" {= "4.02.0+1"} | "camlp4" {= "4.02.0"} | "camlp4" {= "4.01.0"})
> 
> This dependency looks very complex -- it should just be "camlp4" without all
> the versions.

oasis2opam generates this complex dependency because it cannot figure out which versions of the OPAM package camlp4 actually provide the camlp4 library.  If there were "findlib" files in all camlp4 versions in opam-repository, there would not be such long dependency.